_docker/drupal/drupal-filesystem/web/themes/custom/rhd/templates/node--cheat-sheet.html.twig

```diff
@@ -75,7 +75,9 @@ other methods (such as node.delete) will result in an exception.
         <div class="cs-info">
             <ul>
                 <li>
-                    <strong{{ author_attributes }}>By {{content.field_cheat_sheet_author}} </strong>
+                    {% if content.field_cheat_sheet_author|render is not empty %}
```

Comment:
Typically we've also added strip_tags|trim as well as the render. That will guarantee you either have an empty string or not.